src/main/java/com/thealgorithms/conversions/IntegerToEnglish.java

```diff
@@ -0,0 +1,116 @@
+package com.thealgorithms.conversions;
+
+import java.util.HashMap;
+
+public final class IntegerToEnglish {
+    private static final HashMap<Integer, String> BASE_NUMBERS_MAP = new HashMap<>() {
```

Comment:
Will be better to use Map instead of HashMap to declare variable.
And use map method to create:

```
    private static final Map<Integer, String> BASE_NUMBERS_MAP = Map.of(
        0, "", 1, "One", 2, "Two", 3, "Three", 4, "Four", 5, "Five",
        6, "Six", 7, "Seven", 8, "Eight", 9, "Nine", 10, "Ten",
        11, "Eleven", 12, "Twelve", 13, "Thirteen", 14, "Fourteen",
        15, "Fifteen", 16, "Sixteen", 17, "Seventeen", 18, "Eighteen",
        19, "Nineteen", 20, "Twenty", 30, "Thirty", 40, "Forty",
        50, "Fifty", 60, "Sixty", 70, "Seventy", 80, "Eighty", 90, "Ninety", 100, "Hundred"
    );
```

Reply:
Map only allows for a max of 10 key value pair to be passed as arguments. Attached the screenshot below.

<img width="840" alt="image" src="https://github.com/user-attachments/assets/4438e1c1-f02f-460a-8606-c531a806fe41">

Instead using Map.ofEntries() which will not use HashMap

addressed this comment in e792cbcfcb15613c4de642a718854418bc1d5df7